packages/medusa/src/api/routes/store/variants/list-variants.ts

```diff
@@ -132,17 +130,11 @@ export default async (req, res) => {
   const validated = await validator(StoreGetVariantsParams, req.query)
   const { expand, offset, limit } = validated
 
-  let expandFields: string[] = []
-  if (expand) {
-    expandFields = expand.split(",")
-  }
-
   const customer_id = req.user?.customer_id
 
   const listConfig = {
-    relations: expandFields.length
-      ? expandFields
-      : defaultStoreVariantRelations,
+    select: req.listConfig.select,
```

Comment:
Suggestion : req.listConfig already include all of that, we probably can remove the const listConfig entirely